package.json

```diff
@@ -43,62 +43,63 @@
   },
   "devDependencies": {
     "@emotion/jest": "11.11.0",
-    "@emotion/react": "11.11.1",
+    "@emotion/react": "11.11.4",
     "@emotion/styled": "11.11.0",
-    "@formatjs/cli": "6.2.4",
-    "@storybook/addon-a11y": "7.5.3",
-    "@storybook/addon-essentials": "7.5.3",
-    "@storybook/addon-interactions": "7.5.3",
-    "@storybook/addon-links": "7.5.3",
-    "@storybook/addon-themes": "7.5.3",
-    "@storybook/blocks": "7.5.3",
-    "@storybook/manager-api": "7.5.3",
-    "@storybook/react": "7.5.3",
-    "@storybook/react-vite": "7.5.3",
-    "@storybook/testing-library": "0.2.2",
-    "@storybook/theming": "7.5.3",
-    "@swc/core": "1.3.99",
-    "@swc/jest": "0.2.29",
-    "@testing-library/dom": "9.3.3",
+    "@formatjs/cli": "6.2.7",
+    "@storybook/addon-a11y": "8.0.4",
+    "@storybook/addon-essentials": "8.0.4",
+    "@storybook/addon-interactions": "8.0.4",
+    "@storybook/addon-links": "8.0.4",
+    "@storybook/addon-themes": "8.0.4",
+    "@storybook/blocks": "8.0.4",
+    "@storybook/manager-api": "8.0.4",
+    "@storybook/react": "8.0.4",
+    "@storybook/react-vite": "8.0.4",
+    "@storybook/test": "8.0.4",
+    "@storybook/theming": "8.0.4",
+    "@swc/core": "1.4.8",
+    "@swc/jest": "0.2.36",
+    "@testing-library/dom": "9.3.4",
     "@testing-library/jest-dom": "6.1.4",
     "@testing-library/react": "12.1.5",
-    "@testing-library/user-event": "14.5.1",
-    "@tokens-studio/sd-transforms": "0.11.10",
-    "@types/eslint": "8.44.7",
-    "@types/jest": "29.5.10",
-    "@types/jest-axe": "3.5.8",
+    "@testing-library/user-event": "14.5.2",
+    "@tokens-studio/sd-transforms": "0.14.4",
```

Comment:
The last version is 0.15.0 but it relies on an unreleased version of style-dictionary. I figured it would be safer to wait for a style-dictionary to do an official release before updating.